specifications.yml

```diff
@@ -948,6 +1023,33 @@ components:
             type: string
             description: Description
             example: Automatic grant
+    
+    BodyForCustomerApplication:
+      type: object
+      required:
+        - decision
+      properties:
+        decision:
+          $ref: "#/components/schemas/manualDecision"
+        decisionReason:
+          type: string
+          description: Reason to justify the decision taken. (Look at the examples on the right to understand when it's required)
```

Comment:
I feel like the parentheses are redundant, we have examples if they want to but it is enough to just specify what this field is here